src/pages/en/guides/styling.md

```diff
@@ -176,6 +176,168 @@ You can also use the `<link>` element to load a stylesheet on the page. This sho
 
 Because this approach uses the `public/` directory, it skips the normal CSS processing, bundling and optimizations that are provided by Astro. If you need these transformations, use the [Import a Stylesheet](#import-a-local-stylesheet) method above.
 
+## Cascading Order
+
+Astro components will sometimes have to evaluate multiple CSS "instructions" for the same element. For example, your component might import a CSS stylesheet, include its own `<style>` tag, *and* be rendered inside a layout.
```

Comment:
Ooh, I couldn't think of what word I wanted to use instead of "rules" in case that was going to be too repetitive, so I used "instructions" but put in scare quotes to make it a placeholder. Is there a word you like better? Or, if instructions is fine, maybe it should be without the quotation marks?

Reply:
Or just leave it out:

> `.astro` files will sometimes have to evaluate multiple styles that apply to the same element. For example, your component might import a CSS stylesheet, include its own `<style>` tag, *and* be rendered inside a layout.

Reply:
Yup, that looks good. Ship it!

Reply:
I think wherever we use "Astro components", there's a potential for confusion that it doesn't apply to pages, and this is equally true in an intro

Reply:
Following our convo, I'm going to leave it as "Astro components"!